lib/msf/core/exploit/remote/kerberos/client.rb

```diff
@@ -305,44 +305,59 @@ def send_request_tgt(options = {})
             # No etypes specified - how are we supposed to negotiate ciphers?
             raise ::Rex::Proto::Kerberos::Model::Error::KerberosError.new(res: initial_as_res) unless etype_entries
             server_ciphers = etype_entries.decoded_value
+            remaining_server_ciphers_to_attempt = server_ciphers.etype_info2_entries.dup
```

Comment:
Hmm. I think this should be updated to be an intersection of Metasploit's supported ciphers, with the server's supported ciphers.